src/z3c/autoinclude/dependency.txt

```diff
@@ -86,8 +86,8 @@ that it depends on that have ``configure.zcml`` or ``meta.zcml``.
 Note that the individual lists within ``includableInfo`` preserve the
 package order defined in ``setup.py``::
 
-    >>> a_include_finder.includableInfo(['configure.zcml', 'meta.zcml'])
-    {'configure.zcml': ['b.c'], 'meta.zcml': ['z3c.autoinclude', 'testdirective']}
+    >>> list(sorted(list(a_include_finder.includableInfo(['configure.zcml', 'meta.zcml']).items())))
```

Comment:
I ran into problems with `pprint` being switched to `compact=False` in Python 3.4 in another place and worked around it using a `width` parameter, but that's pretty fragile, it seemed to me. It also had issues with the unicode-prefix normalizer. So I wasn't feeling the pprint love :) Plus, I'm not sure...is pprint guaranteed to sort a dictionary?